Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fs2"
-version = "0.4.2"
```

Comment:
Please leave the version bump out, I'll do that upon release.

Reply:
Sure ^^